### PR TITLE
Kathbath asr

### PR DESCRIPTION
**Pull request by bloodraven66**
## What?

asr recipe for KathBath corpus - https://ai4bharat.iitm.ac.in/indic-superb

Todo-
- [x] Train mr model
- [x] change ffmpeg usage
- [x] improve results on some languages
- [x] Upload all models

### CODE REVIEW

File: egs2/kathbath/asr1/local/data.sh [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+if [ -z "${KATHBATH}" ]; then
+    log "Fill the value of 'KATHBATH' of db.sh"
+    exit 1
+fi
+
+download_data="${KATHBATH}"
+
+if [ ${stage} -le 1 ] && [ ${stop_stage} -ge 1 ]; then
+    if [ ! -e "${KATHBATH}/download_done" ]; then
+        echo "stage 1: Data Download to ${LIBRISPEECH}"
```
Comment (ftshijt) [inline egs2/kathbath/asr1/local/data.sh:49]:
```suggestion
        echo "stage 1: Data Download to ${KATHBATH}"
```

File: egs2/kathbath/asr1/local/data.sh [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+                for f in $(find "$path" -type f -name "*$ext")
+                do
+                ffmpeg -loglevel warning -hide_banner -stats -i "$f" -ar 16000 -ac 1 "$f$ext" && rm "$f" && mv "$f$ext" "$f" &
+                done
+
+            done
+        done
+
+
+        # touch "data/dataprep_done"
```
Comment (ftshijt) [inline egs2/kathbath/asr1/local/data.sh:106]:
```suggestion

        touch "data/dataprep_done"
```

File: egs2/TEMPLATE/asr1/db.sh [hunk 3]
```diff
@@ -188,6 +188,7 @@ ITAKO=
 NATSUME=
 KIRITAN=
 NAMINE=
+KATHBATH=/data/Database/Kathbath
```
Comment (sw005320) [inline egs2/TEMPLATE/asr1/db.sh]:
```suggestion
KATHBATH=
```

**Review by sw005320 — COMMENTED** [review]
1. Can you add a result in README.md with pre-trained model links?
2. Can you also add the corpus information here? https://github.com/espnet/espnet/blob/master/egs2/README.md

**Comment by sw005320** [comment]
Please fix CI errors https://github.com/espnet/espnet/actions/runs/5754957584/job/15601353468

**Comment by ftshijt** [comment]
LGTM! Please let us know if you have finished your action items and then we can merge it~ Thanks for your contribution.

**Comment by bloodraven66** [comment]
> LGTM! Please let us know if you have finished your action items and then we can merge it~ Thanks for your contribution.

Sure, should be able to complete in 2 days, will update

**Comment by bloodraven66** [comment]
> LGTM! Please let us know if you have finished your action items and then we can merge it~ Thanks for your contribution.

It is complete

File: egs2/kathbath/asr1/README.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+|malayalam|asr_train_asr_raw_malayalam_bpe500|39.2|30.0|31.3|43.1|
+|bengali|asr_train_asr_raw_bengali_char|17.2|13.2|16.2|21.8|
+|odia|asr_train_asr_raw_odia_bpe500|23.6|16.7|19.9|27.5|
+|gujarati|asr_train_asr_raw_gujarati_char|19.3|15.1|18.4|27.2|
+|hindi|asr_train_asr_raw_hindi_bpe500|12.5|10.1|12.6|14.3|
+|tamil|asr_train_asr_raw_tamil_bpe500|23.3|20.1|24.3|24.2|
+|punjabi|asr_train_asr_raw_punjabi_bpe500|15.9|14.3|14.2|24.9|
+|sanskrit|asr_train_asr_raw_sanskrit_bpe200|40.3|27.7|39.3|49.8|
+|marathi|asr_train_asr_raw_marathi_bpe500|16.6|15.1|16.7|19.7|
+|lang|exp|test|test_known|test_known_noisy|test_noisy|
```
Comment (sw005320) [inline egs2/kathbath/asr1/README.md:17]:
```suggestion
```

**Review by sw005320 — COMMENTED** [review]
Can you fix the conflict?
Also, the README.md looks good, but is it possible to make it more similar (consistent) to the other README.md?

**Comment by bloodraven66** [comment]
> Can you fix the conflict? Also, the README.md looks good, but is it possible to make it more similar (consistent) to the other README.md?

I did not keep the standard readme since there are 12 languages, 4 test set for each language, 12*4*2 rows for WER and CER. Is that ok?

**Comment by bloodraven66** [comment]
> > > Can you fix the conflict? Also, the README.md looks good, but is it possible to make it more similar (consistent) to the other README.md?
> > 
> > 
> > I did not keep the standard readme since there are 12 languages, 4 test set for each language, 12 * 4 * 2 rows for WER and CER. Is that ok?
> 
> Sure, but how about adding the other information, e.g., method name, environment, etc. like https://github.com/espnet/espnet/blob/master/egs2/librispeech_100/asr1/README.md#e-branchformer

Added

**Comment by sw005320** [comment]
Thanks a lot, @bloodraven66!
